src/Bootstrap.php

```diff
@@ -26,12 +26,12 @@ public function setUp(): void
     {
         $this->registerActions();
         $this->registerBlade();
+        $this->enqueueScripts();
     }
```

Comment:
Do you want to add an `enqueueStyles` too, and add the stylesheet to the Vite config for processing? (See above comment about using plain CSS instead of Sass.)

Reply:
That's a good question. Currently when using Vite, we usually import styles in the JS file directly so that wouldn't be required.

Reply:
I think if this is your approach I'd import the stylesheet into the JS file so that it's clear that that's what is intended.

Reply:
Makes sense to me, will add that to the PR.